src/main/java/org/broadinstitute/hellbender/tools/funcotator/DataSourceFuncotationFactory.java

```diff
@@ -169,10 +175,20 @@ public String getVersion() {
         Utils.nonNull(referenceContext);
         Utils.nonNull(featureContext);
 
+        final List<Funcotation> outputFuncotations;
+
         // Query this funcotation factory to get the list of overlapping features.
-        final List<Feature> featureList = queryFeaturesFromFeatureContext(featureContext);
+        // NOTE: This will only get features that are LOCATABLE!
+        //       This corresponds to requiresFeatures() returning `True`.
+        final List<Feature> featureList;
 
-        final List<Funcotation> outputFuncotations;
+        // Only get features if we need them:
```

Comment:
Can you collapse to one line?  `blah? truestuff: falsestuff`

Reply:
Sure.